Cargo.toml

```diff
@@ -158,6 +161,8 @@ wasm-bindgen-futures = { version = "0.4.37", default-features = false }
 wasm-bindgen-test = { version = "0.3.37", default-features = false }
 wasm-bindgen = { version = "0.2.87", default-features = false }
 web-sys = { version = "0.3.64", default-features = false }
+winfsp_build = { version = "0.1.1", default-features = false }
```

Comment:
would have been better to call it `winfsp_wrs_build`

(but you can leave it if it's too complicated to change the name now that a crate has been published)

Reply:
We can't change the name, but yank

Reply:
I leave the decision up to you then ;-)